pinc/Project.inc

```diff
@@ -1266,8 +1267,8 @@ class Project
             // Should really be localized according to the preferences of the *recipient*.
             _("This message is regarding the following project:"),
             "",
-            "\"$title\"",
-            "    $code_url/project.php?id=$projectid",
+            mdmail_append_linebreak("\"$title\""),
```

Comment:
Your changes to this function are what I'd like to see for this PR which makes primarily subtle changes to the email contents.

I'm hoping future iterations, after perhaps some user input, can refine this and other email contents to make them more modern. For instance, I'd love for this to simply be:
```md
[$title]($code_url/project.php?id=$projectid)
```